ocs_ci/templates/ocp-deployment/install-config-aws-ipi.yaml.j2

```diff
@@ -4,7 +4,7 @@ compute:
   - name: worker
     platform:
       aws:
-        type: {{ worker_instance_type | default('m4.large') }}
+        type: {{ worker_instance_type | default('m4.xlarge') }}
```

Comment:
Just thinking about the limit issue we've just hit in AWS about 105 instances. If currently we are using two different and we will switch master and worker to the same it means we can install just 17 clusters right?  @RazTamir I think this will be problematic cause of mentioned issue.

Reply:
Good point,
Let me ask for increase this instance too

Reply:
@petr-balogh 
In this case, do we have a green light to merge this once it's approved and passes deployment testing?

Reply:
based on our discussion I think this PR is ready?